packages/ckeditor5-email/src/integrations/font.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module email/integrations/font
+ */
+
+import { Plugin } from 'ckeditor5/src/core.js';
+import type { FontColorConfig } from '@ckeditor/ckeditor5-font';
+
+import EmailIntegrationUtils from '../emailintegrationutils.js';
+
+/**
+ * A plugin that checks if the Font plugin is properly configured for the email integration.
+ */
+export class FontIntegration extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	public static get requires() {
+		return [ EmailIntegrationUtils ] as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static get pluginName() {
+		return 'FontIntegration' as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static override get isOfficialPlugin(): true {
+		return true;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public afterInit(): void {
+		this._checkColorConfig( 'fontColor' );
+		this._checkColorConfig( 'fontBackgroundColor' );
```

Comment:
Is it necessary to check the color config when the `Font` feature is not available? (And the same for the `Table`?).

Reply:
It's not, as it's highly unlikely that someone used these config options without options, but it'll be better to adjust assertions

Reply:
Fixed in https://github.com/ckeditor/ckeditor5/pull/17849/commits/f28f4f93d23d95adb7135b852f9c3c1a7595c52d